src/Config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <AnalyseParams>
-	<Executable><!-- Required.   -->C:\Program Files (x86)\NUnit 2.6.2\bin\nunit-console.exe</Executable>
+	<Executable><!-- Required.   -->C:\Program Files (x86)\NUnit 2.6.3\bin\nunit-console.exe</Executable>
```

Comment:
I just realized the `restorepackages.bat` script copies the Dynamo Core dependencies into `C:\..\DynamoRevit\src\packages\_packages` and is still pulling the old version of NUnit.  I can probably forget the NuGet changes below and just update the .bat file to retrieve the newer version.  Will push changes to this PR shortly.